addOns/dev/src/main/zapHomeFiles/dev-add-on/tutorial.css

```diff
@@ -19,7 +19,6 @@
 }
 
 .buttonsDiv {
-  position: absolute;
```

Comment:
Had to remove this because at http://localhost:9091/auth/ some of the entries were being covered by the "Index page" button.